plugins/flytekit-duckdb/setup.py

```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=1.3.0b2,<2.0.0", "duckdb", "pandas"]
+plugin_requires = ["flytekit>=1.3.0b2,<2.0.0", "duckdb==1.0.0", "pandas"]
```

Comment:
Does this plugin work with duckdb 0.8 or 0.9? If so, it's better to use `<=` instead
```suggestion
plugin_requires = ["flytekit>=1.3.0b2,<2.0.0", "duckdb<=1.0.0", "pandas"]
```